src/AdminConsole/Pages/App/Playground/NewAccount.cshtml.cs

```diff
@@ -27,7 +27,7 @@ public async Task<IActionResult> OnPostToken(string name, string email)
         var token = await api.CreateRegisterToken(new RegisterOptions()
         {
             UserId = userId,
-            Username = email,

```

Comment:
I'm adding the "Playground: "-prefix to make it clear in browser UI's that this credential is not valid for the AdminConsole, but only for the the Playground (Since they share the same domain).

Otherwise, if you use the playground and create 10 test accounts, the next time you come to the AdminConsole to sign in, you need to figure out which was your test accounts and which was the admin account.